docx/text/tabstops.py

```diff
@@ -21,6 +21,16 @@ class TabStops(ElementProxy):
 
     __slots__ = ()
 
+    def __getitem__(self, idx):
+        """
+        Enables list-style access by index.
+        """
+        tabs = self._element.tabs
+        if tabs is None:
+            raise IndexError
```

Comment:
Each exception we raise should include a meaningful error message to indicate to the developer what went wrong so they know where to look for the problem.

Otherwise, merged just as submitted :)
abab508